pkg/console/operator/operator.go

```diff
@@ -136,10 +143,11 @@ func NewConsoleOperator(
 			configV1Informers.OAuths().Informer(),
 		).WithFilteredEventsInformers( // console resources
 		targetNameFilter,
-		deployments.Informer(),
-		routes.Informer(),
+		deploymentInformer.Informer(),
+		routeInformer.Informer(),
 		serviceInformer.Informer(),
 		oauthClients.Informer(),
+		consolePluginInformer.Informer(),
```

Comment:
are you sure? The events from these informers are filtered so that only changes to resources of name `api.OpenShiftConsoleName` are caught.

The way I understand the API is that there are many objects of the `ConsolePlugin` type that you care about, but I might be wrong.

Reply:
@stlaz right, so we dont need to register the `consolePluginInformer` here ?